src/quo/components/messages/author/style.cljs

```diff
@@ -31,14 +32,13 @@
 
 (defn secondary-name
   [theme]
-  {:padding-top 1
-   :flex-shrink 999999
+  {:flex-shrink 999999
```

Comment:
This one looks weird, do you know what are we trying to achieve?
probably we can remove or change it.

Reply:
I'm not sure what the original intent was, but I think it's meant to ensure that the secondary-name UI will shrink/truncate before anything else. The value seems like it needs to be the highest value (relative to its siblings).